runelite-client/src/main/java/net/runelite/client/plugins/gpu/GpuPlugin.java

```diff
@@ -99,6 +99,9 @@
 	private static final int SMALL_TRIANGLE_COUNT = 512;
 	private static final int FLAG_SCENE_BUFFER = Integer.MIN_VALUE;
 	private static final int MAX_DISTANCE = 90;
+	private static final int SCENE_TILES = 104; // in tiles
```

Comment:
There is a constant in `Constants` for this, which you can just use inline.

Reply:
Thanks!